src/applications/login/components/MockAuthButton.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import environment from 'platform/utilities/environment';
+import ENVIRONMENT_CONFIGURATIONS from 'site/constants/environments-configs';
+
+export default function MockAuthButton() {
+  return (
+    <button
+      type="button"
+      aria-label="Mock Authentication"
+      className="usa-button mauth-button vads-u-margin-y--1p5 vads-u-padding-y--2"
+      onClick={() => {
+        const environ =
+          typeof process !== 'undefined'
+            ? process.env.BUILDTYPE
+            : environment.BUILDTYPE;
+        window.location.replace(
+          `${
+            ENVIRONMENT_CONFIGURATIONS[environ].API_URL
```

Comment:
Take a look at the environment utilities in `platform/utilities/environment/index.js`

Reply:
Funny enough, the original version of this used the `API_URL` from that file. Unfortunately, I couldn't use that in conjunction with the tests I wrote (which change `process.env.BUILDTYPE` to see if the URL gets set properly in different environments). This was the best way I could think of to have the button work both during the tests and when deployed. If there's a better way, please feel free to let me know!